README.md

```diff
@@ -8,49 +12,62 @@ It is assumed you have the requirements for doing basic node.js development alre
 npm i
 ```
 
-Latest Local DWS REST API Documentation as of May 5th, 2023.
+Latest Local DWS REST API Documentation as of July 20, 2023.
 [Local DWS APIs](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/1172734089/Local+DWS+APIs)
 
+## Create players.json
+
+You will need to create the players.json file. This can be done by deleting the '.example' from the end of the file in cli-src, or by creating a new players.json file in the same folder. Once this file is created, and the module is installed, you can populate this object yourself, or use the command line:
+```
+bsc addPlayer <playerName> <ipAddress> [username] [password]
+```
+
+**Make sure to install the CLI before using the above command to edit players.json.**
+
 ## Installing NPM Module from source
 
 The following will need to be run to build the npm module. `bsc` will be built and ready to use. 
 
 ```bash
 npm -g install
-bsc
 ```
 
+
 ## Usage
 
-Note: By default, `bsc` will return the response `{data: {result: {...}}}`. To retrieve the raw HTTP Response add the `-a=true` argument.
+For a list of commands and their usage, use `bsc --help`.
 
-Example 1:
+This CLI uses a player configuration object to interact with your player(s). This configuration can be found in the players.json file currently. You can add and remove players from this object at the command line: 
 ```bash
-bsc -i=192.168.128.101 -p=ABC01A000001 -m=GET -r="/info"
+bsc addPlayer [playerName] [ipAddress] [lDWS username] [lDWS password]
+bsc rmPlayer [playerName]
 ```
-
-Example 2:
-```bash
-bsc -i=192.168.128.101 -p=ABC01A000001 -m=GET -r="/files/sd"
+### Features
+
+| Feature | Implemented? | Function | Usage |
+| --------- | ---- | ------------------------ | -------------------------------------- |
+| addPlayer | [x] | Add player configuration to players.json | bsc addPlayer <playerName> <ipAddress> [lDWS username] [lDWS password] |
+| rmPlayer | [x] | Remove player configuration from players.json | bsc rmPlayer <playerName> |
+| editPlayer | [] | Edit an already existing player's info |  |
+| getDI | [x] | Get device info in the form of a JSON object | bsc getDI <playerName> | 
+| put | [x] | Put file/files on a player. Specify a single file or a directory of files and the upload location (optional) | bsc put <playerName> <File/Directory> [location] |
+| reboot | [x] | Reboot the specified player | bsc reboot <playerName> |
+| checkPW | [x] | Check if lDWS password is enabled | bsc checkPW <playerName> |
+| screenshot | [x] | Take a screenshot | bsc screenshot <playerName> |
+| delFile | [] | Delete a file on the player |  | 
```

Comment:
If an API or feature has yet to be developed, should we take this internal as a feature request?

Reply:
Yeah, makes sense to me. That being said, all of the 'not implemented' features on that list should be supported by already existing API's.